client/me/memberships/subscription.jsx

```diff
@@ -135,13 +136,14 @@ function Subscription( { translate, subscription, moment, stoppingStatus } ) {
 					</Card>
 					<CompactCard
 						tagName="button"
-						className="memberships__subscription-remove"
```

Comment:
If we're removing this class, we should remove its implementation also (AFAIK this is the only place it's used).

https://github.com/Automattic/wp-calypso/blob/261fc54a839004ba261c9269130782a7f4d52942/client/me/memberships/subscription.scss#L119